src/scss/_mixins.scss

```diff
@@ -54,3 +54,25 @@
 	width: $size;
 	height: $size;
 }
+
+@mixin oLoadingContent($opts: (
+	'color': null,
+	'size': null
+)) {
+	$color: map-get($opts, 'color');
+	$size: map-get($opts, 'size');
+
+	@include _oLoadingAnimationKeyframes();
```

Comment:
Hmm I think we're going to have to reintroduce the "hack" and use `@root` here 😞. Because the content mixin is going to be called within a class.